doc/modules/ROOT/pages/canceling.adoc

```diff
@@ -0,0 +1,50 @@
+= Canceling Client Operations
+
+Some of Hazelcast .NET client operations support cancellation via a `CancellationToken`. This section details what actually happens when that token is cancelled.
+
+== Default Operations
+
+When a `CancellationToken` is passed to a method that triggers a server-side operation, the token is ultimately passed to the internal method which sends the request message to the server. At that point:
+
+* If the token is cancelled before the request has been sent, the request is not sent
+* If the token is cancelled while retrying to send a request, the retry is aborted
+
+Otherwise, the request message is sent as a whole non-cancellable operation, and the client then waits for a response message from the server. It the token is cancelled during this wait, the wait is aborted and the response message will be ignored when it is received.
```

Comment:
```suggestion
Otherwise, the request message is sent as a whole non-cancellable operation, and the client then waits for a response message from the server. If the token is cancelled during this wait, the wait is aborted and the response message will be ignored when it is received.
```